public/static/docs/api-reference/get_url.md

```diff
@@ -41,39 +50,31 @@ an appropriate library, such as
 or
 [`paramiko`](https://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.get).
 
-Note that if the target is a directory, the URL will end in `.dir`, as DVC
-stores a special JSON file with `.dir` extension that contains the mapping of
-files in the directory (as a JSON array), along with their hash values. Refer to
-[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
-and `dvc add` to learn more about how DVC handles data directories.
-
 ## Parameters
 
 - **`path`** - location and file name of the file or directory in `repo`,
   relative to the project's root.
 
 - `repo` - specifies the location of the DVC project. It can be a URL or a file
-  system path. Both HTTP and SSH protocols are supported for online Git repos
-  (e.g. `[user@]server:project.git`).
-
-  The current project is used by default if a `repo` argument is not given (the
-  current working directory tree is walked up to find it).
+  system path. Both HTTP and SSH protocols are supported for online Git
+  repositories (e.g. `[user@]server:project.git`). _Default_: The current
+  project is used (the current working directory tree is walked up to find it).
```

Comment:
Question: Is "Deault:" and "...if argument is not given" the same? I guess so but perhaps it's confusing to call this _default_ when the signature shows `None` is the default value? And is "Default:" in the same p better or clearer than a separate paragraph? I'm not sure.

Again no strong opinion here. Up to you but we should update the params in all the API refs.

Reply:
p.s. https://github.com/iterative/dvc.org/pull/1032/files#r386754413

Reply:
p.s. https://github.com/iterative/dvc.org/pull/1032#pullrequestreview-367651464

Reply:
I think it's clear .. also I think you used the phrase by default. I didn't change the meaning - just made it shorter. I'm fine to keep them each on a separate line, but with some common structure and highlighting _Default_: or something else

Reply:
No, its OK. I'm convinced here now. Resolving.